src/convert.ts

```diff
@@ -13,7 +13,7 @@ export function convert(options: ProjectOptions, debugLogging: boolean = false)
   project.getSourceFiles().forEach(sourceFile => {
     const filePath = sourceFile.getFilePath();
     if (debugLogging) {
-      console.log(`Checking: ${filePath}`);
+      console.log(` Checking: ${filePath}`);
```

Comment:
What is the purpose of this "space" character?

Reply:
The space is so that you get results like:
```log
Processing: .\tsconfig.json
 Checking: D:/GitHub/ts2esm/src/cli.ts
 Checking: D:/GitHub/ts2esm/src/convert.ts
  Modified (🔧): D:/GitHub/ts2esm/src/convert.ts
 Checking: D:/GitHub/ts2esm/src/parseInfo.ts
 Checking: D:/GitHub/ts2esm/src/util.test.ts
 Checking: D:/GitHub/ts2esm/src/util.ts
Processing: .\tsconfig.json
 Checking: D:/GitHub/ts2esm/src/cli.ts
 Checking: D:/GitHub/ts2esm/src/convert.ts
```

Reply:
Cool idea. Shall we use tabs for the indentation?

Reply:
That would be your call. I think a single space per is good enough. Some paths can be long.

Reply:
Ok, let's go with 1 then. 👍